.gitignore

```diff
@@ -16,6 +16,7 @@
 *.launch
 .settings/
 *.sublime-workspace
+.vscode/
```

Comment:
can we gitignore this and remove the `.vscode` files? I guess they may have been added by accident?

Reply:
Sure :)